spring-boot-project/spring-boot-actuator/src/test/java/org/springframework/boot/actuate/cassandra/CassandraHealthIndicatorTests.java

```diff
@@ -47,25 +48,28 @@ void createWhenCassandraOperationsIsNullShouldThrowException() {
 
 	@Test
 	void healthWithCassandraUp() {
-		CassandraOperations cassandraOperations = mock(CassandraOperations.class);
-		CqlOperations cqlOperations = mock(CqlOperations.class);
-		CassandraHealthIndicator healthIndicator = new CassandraHealthIndicator(cassandraOperations);
-		given(cassandraOperations.getCqlOperations()).willReturn(cqlOperations);
-		given(cqlOperations.queryForObject(any(SimpleStatement.class), eq(String.class))).willReturn("1.0.0");
+		CqlSession session = mock(CqlSession.class);
+		ResultSet resultSet = mock(ResultSet.class);
+		Row row = mock(Row.class);
+		given(session.execute(any(SimpleStatement.class))).willReturn(resultSet);
+		given(resultSet.one()).willReturn(row);
+		given(row.getString(0)).willReturn("1.0.0");
+		CassandraHealthIndicator healthIndicator = new CassandraHealthIndicator(session);
 		Health health = healthIndicator.health();
 		assertThat(health.getStatus()).isEqualTo(Status.UP);
 		assertThat(health.getDetails().get("version")).isEqualTo("1.0.0");
 	}
 
 	@Test
 	void healthWithCassandraDown() {
-		CassandraOperations cassandraOperations = mock(CassandraOperations.class);
-		given(cassandraOperations.getCqlOperations()).willThrow(new CassandraInternalException("Connection failed"));
-		CassandraHealthIndicator healthIndicator = new CassandraHealthIndicator(cassandraOperations);
+		CqlSession session = mock(CqlSession.class);
+		given(session.execute(any(SimpleStatement.class)))
+				.willThrow(new DriverTimeoutException("Connection timed out (not really)"));
```

Comment:
Please change that to `Test Exception` if you want to make it clear it's a test exception.